### ISSUE TITLE
Gaming site URL is not loading

### ISSUE THREAD

**Issue by nrajan002c** [Metrological/WebKitForWayland#617]
The below url is consistently fails to load from the initial launch screen in wpe-2.22 and wpe-20170728 branch.
http://nxg-frontend.uphe.com/6F47-7AF8-4CCE-9094-F86D-4/Code/FOD/B2AB2F6B-0A06-EBD7-77CE-A18964E93A76/GidgetsCatTrap/#0524DE4A-AEBD-BEDF-647F-E1FD6B446566

But it is loading fine in wpe webkit trunk and Chrome browser.
Can you please check it.

**Comment by magomez** [comment]
@nambirajang discovered that the problem here was the lack of support of jpeg2000. I backported the patch that adds support for it from upstream to the wpe-20170728 and wpe-2.22 branches and checked that it fixes the issue. I pushed the changes, but kept the feature disabled, as it requires a new dependency (openjpeg >= 2.2.0) that not every environment may have, so it would break the build in those cases. Passing -DUSE_OPENJPEG=ON during the build will enable the feature.
